RDF-Pipeline/t/tests/0049_DependsOn_clock_site/setup-files/node/pipeline.ttl

```diff
@@ -5,5 +5,5 @@
 @prefix : <http://localhost/node/> .
 
 :hello a p:FileNode ;
-  p:dependsOn <http://www.time.gov/timezone.cgi?Eastern/d/-5> .
+  p:dependsOn <http://www.time.gov> .
```

Comment:
OK.  Any thoughts on where we can find a URL that would emulate what you were doing before?  The one you were using simply no longer exists.